ursula_cli/shell.py

```diff
@@ -104,6 +104,8 @@ def run(args, extra_args):
     if not os.path.exists(args.environment):
         raise Exception("Environment '%s' does not exist", args.environment)
 
+    _append_envvar('URSULA_ENV', os.path.abspath(args.environment))
```

Comment:
if URSULA_ENV already exists won't this append to it ?  so a person might end up with it being `/home/craig/ursula/env/potato/home/craig/ursula/env/potato`